cpu/saml21/periph/pm.c

```diff
@@ -38,6 +38,7 @@ void pm_set(unsigned mode)
                 DEBUG("pm_set(): setting STANDBY mode.\n");
                 _mode = PM_SLEEPCFG_SLEEPMODE_STANDBY;
                 break;
+            default:
```

Comment:
This should be in a separate PR (gcc7 complaining?)

Reply:
Yes, it is in #9244. As soon as that is merged, I will rebase and that change should disappear.

Reply:
ok!